app/workers/vbms/submit_dependents_pdf_job.rb

```diff
@@ -7,7 +7,7 @@ class Invalid686cClaim < StandardError; end
     include SentryLogging
 
     # Generates PDF for 686c form and uploads to VBMS
-    def perform(saved_claim_id, va_file_number_with_payload)
+    def perform(saved_claim_id, va_file_number_with_payload, submittable_686, submittable_674)
```

Comment:
Consider using keyword arguments, especially with longer method signatures as it makes it easier to understand what the function requires as input params.

Reply:
thank you @kfrz, wanted to let you know I updated this to use keyword arguments 👍